etna/ciim/client.py

```diff
@@ -89,12 +93,27 @@ def format_list_param(items: Optional[list]) -> Optional[str]:
 
     This function, when given a list, will return a comma-separated string.
 
-    Additionally, to overcome an issue with the API, any commas within paramter
-    text will be removed.
+    Additionally, to overcome an issue with the API, any commas within
+    paramter text will be removed.
     """
     if not items:
         return None
-    return ", ".join([i.replace(",", "") for i in items])
+
+    if param_type == "filter_aggregations":
+        # reconstruct the required starting value that contains special value
+        # replace special chars by space
+        regex = r"([():\\/,&])"
+        subst = " "
+        items = [
+            i.split(":", 1)[0]
+            + ":"
+            + re.sub(regex, subst, i.split(":", 1)[1], 0, re.MULTILINE)
+            for i in items
+        ]
+    else:
+        items = [i.replace(",", "") for i in items]
```

Comment:
With lists of values represented as `?param=1&param=2&param=3`, commas in values should no longer be an issue, so we can turn this function into a dedicated `prepare_filter_aggregations()` function.

Reply:
Done